### ISSUE TITLE
Save Manager DOESN'T work on physical (bought) games?

### ISSUE THREAD

**Issue by Dochartaigh** [d3m3vilurr/vita-savemgr#37]
Can you please confirm that Save Manager ONLY works on backing up/restoring "backup" games (i.e. Mai or Vitamin dumps)? I tried to backup two physical games I own, and run off the Sony Vita game cartridges (i.e. the actual game you would buy from GameStop or similar), and I'm getting an "Cannot find eject data" error when loading either game after backing up the saves.

It seems to work just fine on my Mai dumps.

**Comment by Konnoke** [comment]
Save Manage works on both digital (mai/vitamin/legit) and physical games. At one point, around 0.5, it only supported physical and unencrypted games such as vitamin/mai dumps but later versions added support for encrypted digital games.

Which version of savemgr did you last used before you got the error? I used to get the same "Cannot find eject data" errors from an old version of save manager back when it did not support encrypted digital games.

**Comment by Dochartaigh** [comment]
I downloaded it from the Github a couple hours ago:

https://github.com/d3m3vilurr/vita-savemgr/releases/

I honestly don't know if I clicked on the nightly, or the release 0.8.0. I'm going to download the stable 0.8.0 right now and try it again.

The two games in question are Final Fantasy X HD Remaster, and Metal Gear Solid HD if that matters. I'll report back after I try 0.8.0.

**Comment by Dochartaigh** [comment]
Confirmed: It was the nightly that has the problem. 0.8.0 seems to be working fine (and I even re-downloaded and installed the same nightly .VPK I downloaded earlier today to rule out a .VPK installation error from vitaShell or anything like that).

Question though: so I've tried this like 10+ times now to troubleshoot, right? The last time I had 1 single save game on the 2nd room of Metal Gear Solid (MGS). When I delete the MGS game bubble again, restore Slot 0's save data (triangle button=import option), I now have TWO save games showing up in MGS. One seems to be leftover from an earlier backup I made through Save Manager (where I'm in the very first room of the game), and the other save is the correct most-recent save I just saved like 2 minutes ago.

I literally deleted everything again (down to the level of re-downloading the 0.8.0 NON-nightly and FTP'ing it over to the VIta again, rebooting between many of these steps even), and have duplicated this same weirdness.

Does Vita Save Manager merge save games together if there's multiple save game slots (like most games use)? Everything I've read says it takes EVERYTHING (i.e. ALL save states) for that select game and will back them up or restore them in whole - NEVER merging them.
